pkg/operator/rbac.go

```diff
@@ -59,7 +59,7 @@ func MakeRBACObjects(name, namespace, component string, additionalRules []rbacv1
 func MakeScopedRBACObjects(fbName, fbNamespace string, saAnnotations map[string]string) (*rbacv1.Role, *corev1.ServiceAccount, *rbacv1.RoleBinding) {
 	r := rbacv1.Role{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "kubesphere:fluent",
+			Name:      "fluent-operator:fluent",
```

Comment:
`fluent:fluent-operator` is more appropriate.

Reply:
agree